samples/bluetooth/central_bas/src/main.c

```diff
@@ -119,8 +120,23 @@ static struct bt_gatt_dm_cb discovery_cb = {
 	.error_found = discovery_error_found_cb,
 };
 
+static void gatt_discover(struct bt_conn *conn)
+{
+	if (conn == default_conn) {
```

Comment:
For consistency we should change this to:
```
if (conn != default_conn) {
    return;
}

int err = bt_gatt ....
```

EDIT:
I see that it was already like this and that this is not new code you wrote, also there are other instances of the same code.
I would however still like to do this change, if you don't mind.

Reply:
This is fixed now